### ISSUE TITLE
How to add  far-star?

### ISSUE THREAD

**Issue by francesco87** [FortAwesome/react-fontawesome#336]
How can I show in my react project "far-star" icon (empty star icon with only borders)?

**Comment by robmadole** [comment]
Hi @francesco87 what have you tried so far?

**Comment by francesco87** [comment]
importing '@fortawesome/free-solid-svg-icons' I found faStar and faStarHalfAlt but I can't found farStar it is the empty start with only border.

**Comment by robmadole** [comment]
The regular style of the `star` icon is available from the `@fortawesome/free-regular-svg-icons` package. Try that and see if you can find what you need.

**Comment by richardsondx** [comment]
+1 instructions are not clear for far icons

**Comment by alice-0-kim** [comment]
+1 towards mending documentation although the package name itself is self-explanatory

**Comment by MorenoMdz** [comment]
Just adding to this, I feel the icons library that can be used is not clear at all. Docs need a bit more info in which icons can be used and provide links to its gallery if possible.

**Comment by mohdafzal330** [comment]
No solution provided yet ?

**Comment by robmadole** [comment]
@mohdafzal330 what have you tried that isn't working?

**Comment by mohdafzal330** [comment]
> @mohdafzal330 what have you tried that isn't working?

I did find any icon in react similar to **fa-star-o**

**Comment by robmadole** [comment]
@mohdafzal330 

```
import { faStar } from '@fortawesome/free-regular-svg-icons'
```

**Comment by mohdafzal330** [comment]
@robmadole while importing **faStar** , it gives us filled star, But I need empty star

**Comment by francesco87** [comment]
solved

**Comment by mohdafzal330** [comment]
Let me check

**Comment by DAVIS-PYTH** [comment]
@francesco87 how did you solve that

**Comment by joshharper1** [comment]
> @francesco87 how did you solve that

Apologies if this has been answered but here is how I managed to work around this issue. 

I used npm to install 'npm i --save @fortawesome/free-regular-svg-icons' & 'npm i --save @fortawesome/free-solid-svg-icons'. I then created a library so I could have two variations of one icon (faStar) and added both of them like so;

import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { library } from '@fortawesome/fontawesome-svg-core'
import { faStar as fasFaStar } from '@fortawesome/free-solid-svg-icons'
import { faStar as farFaStar } from '@fortawesome/free-regular-svg-icons'
import { faStarHalfAlt} from "@fortawesome/free-solid-svg-icons";
library.add(fasFaStar, farFaStar, faStarHalfAlt)

Hope this helps.
